### DISCUSSION TITLE
Possibility to select and copy information from notes for read only user

### DISCUSSION THREAD

**Discussion by andy1463** [1Remote/1Remote#513] [Ideas]
Hello,
would be great if a user which has read-only permission on sql database could select and copy information from notes for a remote connection.
It is only possible with full permission.

Thanks and best regards

**Comment by VShawn** [comment]
I'm not sure if it's possible to enable selecting and copying in read-only mode for notes, as the markdown display component is developed by a third party. However, I can add a copy button that allows you to copy the entire note.

**Comment by andy1463** [comment]
I am using notes for different information. Usually I only need a part of the note to be copied. So copying of entire note is not useful in most cases because I have to edit the note before I can use the part I need. In most cases I think it is more easy to write down the part of the note than copy and edit the whole note.

**Reply by majkinetor** [reply]
@VShawn, we can maybe provide an option for note to be switched to Markdown text in regular selectable text box for the purpose of copying.

**Reply by VShawn** [reply]
just a workaround, I can make it auto switch to readonly markdown source code by mouse hover, user can select and copy text from there.

**Reply by andy1463** [reply]
That would be great

**Comment by VShawn** [comment]
close because of #550
